docs/model_configuration.md

```diff
@@ -501,6 +501,59 @@ the device kind of the instance, for instance, KIND_CPU is "cpu", KIND_MODEL is
   ]
 ```
 
+### Rate Limiter Config
+
+Instance group optionally specifies [rate limiter](rate_limiter.md)
+config which controls how the rate limiter operates on the instances
+in the group. The rate limiter configuration includes the following
+specifications:
+
+#### Resources
+
+The set of resources required to execute a model instance. Essentially,
```

Comment:
Change "resources" into a link that points to the section in rate limiter section. I don't think you need anything after "Essentially" as you should explain that in rate limiter section (see my comments there). But you should say that the count refers to the number of copies of the resource that the model instance requires to run (see below for other comments on careful use of wording).